package Exceptionhandling;

import java.util.Scanner;

public class Exceptiondemo {

	public static void main(String[] args) {
		System.out.println("programing is started .......");
		Scanner sc=new Scanner (System.in);
		
/*		
		//example1
		int num=sc.nextInt();
		System.out.println(100/num);   //arithmetic Exception 

		
		//Example2 
		int a[]= new int[5];
		System.out.println("Enter the position (0-4) ");
		int pos=sc.nextInt();
		
		System.out.println("enter a value ");
		int value=sc.nextInt();
		
		a[pos]=value;
		
		System.out.println(value);
		

		
		//Example3
		String s="welcome";
		int num=Integer.parseInt(s);
		System.out.println(num);
*/		
	
		//example 4
		
		String s =null;
		System.out.println(s.length());
		
		
		
		
		
		
		
		System.out.println("program is completed ");
		System.out.println("program is exited");
		
	}

}
